BuildScript/BuildTargets/Testing.targets

```diff
@@ -36,25 +36,19 @@
 
   <ItemGroup>
     <NumberOfTestFailuresPerProject Include="0"/>
-    <NumberOfTestErrorsPerProject Include="0"/>
   </ItemGroup>
 
   <!-- Main target; reporting only, calls RunTestsInternal for actual work -->
   <Target Name="RunTests" DependsOnTargets="CreateTestConfigurations;CreateLogDirectory;BuildTestProjects;LogRunTestsStartTime;RunTestsInternal">
     <Error Text="The property 'ConfigurationID' is not set." Condition="'$(ConfigurationID)' == ''" />
     <Error Text="ItemGroup 'NumberOfTestFailuresPerProject' is empty, but should contain test results!" Condition="@(NumberOfTestFailuresPerProject) == ''"/>
-    <Error Text="ItemGroup 'NumberOfTestErrorsPerProject' is empty, but should contain test results!" Condition="@(NumberOfTestErrorsPerProject) == ''"/>
```

Comment:
is the errors-concept gone in NUnit 3? Please link docs for review/know-how building

Reply:
I just noticed that the `errors` attribute was gone from the result xml. Assertion failures and unexpected exceptions (in tests and setup/teardown methods) now seem to all be grouped as failures (according to the things I tried).

[This](https://github.com/nunit/docs/wiki/XML-Formats) and [this](https://github.com/nunit/docs/wiki/Test-Result-XML-Format) explain the new format. It says that an `<assertion>` tag has an attribute `result` that may be  Inconclusive, Passed, Warning, Failed or Error, but it looks like we can't just easily get the number of errors and failures (which is probably fine since all we did was group them anyway)

Reply:
Okay, good to know. Please try to trigger somehting that resulted in an error in nunit2 (if reasonably possible) and then try this again in nunit3 and nunit3 with nunit2-compat to see if the failure count covers everything or if we need to improve the error detection.

Reply:
Tested in IO: 3 errors and 2 failures in V2 with the old runner were always 5 failures with the new runner, with v2 compatibility and without.

Looks like we're fine